rescript.json

```diff
@@ -1,4 +1,4 @@
 {
   "name": "rescript",
-  "dependencies": ["@tests/gentype-react-example"]
+  "dependencies": ["@tests/gentype-react-example", "playground"]
```

Comment:
`"playground"` is part of the workspace, so I need to be listed here for rewatch to pick up the correct `node_modules` folder.

The `package.json` has `rescript clean` in the commands.
This will now accurately only clean the playground package.

But it needs that link to the parent rescript json.